docs/docsite/rst/reference_appendices/faq.rst

```diff
@@ -643,6 +643,34 @@ but you can still access the original via ``hostvars``::
 This works for all overriden connection variables, like ``ansible_user``, ``ansible_port``, etc.
 
 
+.. _scp_protocol_error_filename:
+
+How do I fix 'protocol error: filename does not match request' when fetching a file?
+++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++
+
+When using ``scp`` as the file transfer mechanism for SSH, it can fail with the error::
+
+    failed to transfer file to /tmp/ansible/file.txt\r\nprotocol error: filename does not match request
+
+This is due to a bug in newer releases of OpenSSH in the ``scp`` client that is used on the Ansible controller. These
+releases attempt to validate that the path of the file to fetch matches the requested path. This validation
+will fail if the remote filename requires quotes to escape spaces or non-ascii chars in its path. If you come across
+this error there are a few different ways to ignore this validation, such as:
+
+* Don't use the ``scp`` transfer mechanism by
```

Comment:
Does this mean "avoid using `scp`, here's how" or "avoid these particular methods of using `scp`"? I'd like to rephrase this to be unambiguous and positive ("You will trigger this error if you:" or say "avoid" rather than "don't", depending on the correct meaning here).